### PR TITLE
Adding more scenarios for day1 operation

### PR DESCRIPTION
**Pull request by varshab1210**
**What type of PR is this?**

/kind enhancement

**What does this PR do / why we need it**:

Added scenario for testing KAM bootstrap command with and without --overwrite flag. This pr also fixes the need to generate resource files with different names by kam bootstrap command (which are redundant) by cleaning up the directory before each scenario

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #? N/A

**How to test changes / Special notes to the reviewer**:

```make e2e``` should pass

### CODE REVIEW

File: test/e2e/kamsuite/kamsuite.go [hunk 1]
```diff
@@ -139,6 +144,24 @@ func deleteGitlabRepository(arg []string) (bool, string) {
 	return true, stderr.String()
 }
 
+func ClearDir() error {
```
Comment (chetan-rns) [inline test/e2e/kamsuite/kamsuite.go:147]:
```suggestion
func clearDir() error {
```
The function is exported but only used within the same package

File: test/e2e/kamsuite/kamsuite.go [hunk 2]
```diff
@@ -64,6 +65,10 @@ func FeatureContext(s *godog.Suite) {
 
 	s.AfterScenario(func(*messages.Pickle, error) {
 		fmt.Println("After scenario")
+		err := ClearDir()
```
Comment (chetan-rns) [inline test/e2e/kamsuite/kamsuite.go:68]: If we are trying to delete the gitops folder `os.RemoveAll(path)`  will remove the entire folder. WDYT?

**Review by chetan-rns — CHANGES_REQUESTED** [review]

File: test/e2e/kamsuite/kamsuite.go (hunk 2 above)
Comment (varshab1210) [reply test/e2e/kamsuite/kamsuite.go:68]: I had tried it that way but it would remove even ```test-run``` folder and cause test failure.

Comment (varshab1210) [reply test/e2e/kamsuite/kamsuite.go:68]: Thanks for the advice @chetan-rns , will do

**Review by chetan-rns — APPROVED** [review]
Thanks @varshab1210 !

/lgtm
/approve

**Pull request merged**